### DISCUSSION TITLE
Rust board group buy? Or just use Esp32 c3 mini

### DISCUSSION THREAD

**Discussion by tomasApo** [esp-rs/esp-rust-board#47] [General]
Is anyone selling the rust boards? 

Doing some rough calculations using PCBway, if we do a group buy the price goes down significantly per board. 2 boards = $70 (each)

50 boards = 5$ (each)

Is there a significant difference when following the tutorial book between esp32-c3 mini dev board and the rust board?
https://www.mouser.co.uk/ProductDetail/Espressif-Systems/ESP32-C3-DevKitM-1?qs=pUKx8fyJudB1sOWbbEnGFw%3D%3D 

![ESP32-C3-DevKitM-1_SPL](https://user-images.githubusercontent.com/75183079/173932336-e761ac6b-3db4-46b6-a340-3d73d86db888.jpg)

**Comment by MabezDev** [comment]
Mass production of the board is underway, but due to various hold-ups, namely the lockdown in Shanghai, it's later than we would have liked. Of course, we provided the board schematics and BOM so anyone is welcome to do a group buy run to speed things up :).

**Comment by pedrominatel** [comment]
Hi,

You can buy at [Mouser](https://www2.mouser.com/ProductDetail/Espressif-Systems/ESP32-C3-DevKit-RUST-1?qs=4ASt3YYao0WvXOj9TGjU2A%3D%3D) or [AliExpress](https://pt.aliexpress.com/item/1005004418342288.html?spm=a2g0o.store_pc_topSellerIng.8148356.4.7acd6e1cGqW0UE&pdp_npi=2%40dis%21EUR%21%E2%82%AC%2024%2C38%21%E2%82%AC%2024%2C38%21%21%21%21%21%40210318b916703270427498546ebf06%2112000029115522071%21sh&gatewayAdapt=glo2bra).

**Reply by tomasApo** [reply]
Obrigado Pedro, 
Bought one last month

**Reply by pedrominatel** [reply]
Sorry for the late reply.

**Comment by dantho** [comment]
I've purchased 3 from [Mouser](https://www2.mouser.com/ProductDetail/Espressif-Systems/ESP32-C3-DevKit-RUST-1?qs=4ASt3YYao0WvXOj9TGjU2A%3D%3D) for $21 each.  If you order, be prepared for your order to get stuck at Export Control until you sign and return an _End User Certification_ document they email you separately.  Sheesh.

My 3 boards are chatting with each other using esp-now from the very helpful examples in the bare-metal [esp-rs/esp-wifi](https://github.com/esp-rs/esp-wifi) repo.
